### ISSUE TITLE
Unable to open tiff files with the Ilastik 1.2orc10

### ISSUE THREAD

**Issue by MartaFrat** [ilastik/ilastik#1338]
Hi,

I am unable to open my tiff files using the latest ilastik version.
I am running it on mac os x (10.11.6) (Retina, late 2013).
I have attached the log file for your reference and look forward to a speedy resolution of the bug.

Cheers,
Marta
[Ilastik_log copy.docx](https://github.com/ilastik/ilastik/files/459547/Ilastik_log.copy.docx)

P.S.: the tiff files can be opened using the orc5 version of ilastik (CPLEX installed). However, I am unable to export the files after tracking with learning procedure.

---

Edit: Here's the error log:

``` pytb
$ tail -f ilastik_log.txt 
    self._configureOperator(self)
  File "/Applications/Ilastik/ilastik-1.2.0rc10-OSX.app/Contents/ilastik-release/ilastik-meta/lazyflow/lazyflow/slot.py", line 1415, in _configureOperator
    self.operator._setupOutputs()
  File "/Applications/Ilastik/ilastik-1.2.0rc10-OSX.app/Contents/ilastik-release/ilastik-meta/lazyflow/lazyflow/operator.py", line 450, in _setupOutputs
    self.setupOutputs()
  File "/Applications/Ilastik/ilastik-1.2.0rc10-OSX.app/Contents/ilastik-release/ilastik-meta/lazyflow/lazyflow/operators/ioOperators/opTiffReader.py", line 84, in setupOutputs
    assert N == 1, "Don't know how to handle any number of planes except 1 (per page)"
AssertionError: Don't know how to handle any number of planes except 1 (per page)

ERROR 2016-09-06 11:08:49,444 log_exception 793 140735167401984 Wasn't able to load your dataset into the workflow.  See error log for details.

```

**Comment by stuarteberg** [comment]
Thanks for the report.  Can you attach a sample .tiff file here (or email it to team@ilastik.org)?

**Comment by MartaFrat** [comment]
Yes i ll send it to the email address you wrote me,
Thanks
Marta

**Comment by stuarteberg** [comment]
Thanks for the sample file.  I'm able to reproduce the problem.  What are the dimensions of this image?  Is it 195 time steps, 512*512 (XY) and 2 channels?  What software did you use to generate this file?

**Comment by MartaFrat** [comment]
Yes it s exactly like you wrote; the software is Volocity, i don remember now which version it is but if you need it i can tell you asap .

**Comment by stuarteberg** [comment]
This may not be a quick fix, and I am leaving for an extended vacation this afternoon.

If you are able to convert the file into a different format (such as a sequence of 2D tiffs or pngs, or OME-TIFF, or HDF5), that may be easiest.  If an older version of ilastik was able to read this file, you can use ilastik itself to convert the file.  (Older versions of ilastik can be downloaded from here: http://files.ilastik.org/).

**Comment by ilastikdev** [comment]
Opening Marta's file I get the following error in the current master:

ERROR 2016-09-21 14:51:48,988 log_exception 18336 140464270169920 Wasn't able to load your dataset into the workflow.  See error log for details.
ERROR 2016-09-21 14:52:32,265 log_exception 18336 140464270169920 Traceback (most recent call last):
  File "/home/jales/software/miniconda/envs/ilastik-devel/ilastik-meta/ilastik/ilastik/applets/dataSelection/dataSelectionGui.py", line 624, in _configureOpWithInfos
    self.topLevelOperator.DatasetGroup[laneIndex][roleIndex].setValue( info )
  File "/home/jales/software/miniconda/envs/ilastik-devel/ilastik-meta/lazyflow/lazyflow/slot.py", line 110, in call_in_setup_context
    return func(self, *args, **kwargs)
  File "/home/jales/software/miniconda/envs/ilastik-devel/ilastik-meta/lazyflow/lazyflow/slot.py", line 1195, in setValue
    self._changed()
  File "/home/jales/software/miniconda/envs/ilastik-devel/ilastik-meta/lazyflow/lazyflow/slot.py", line 1395, in _changed
    c._changed()
  File "/home/jales/software/miniconda/envs/ilastik-devel/ilastik-meta/lazyflow/lazyflow/slot.py", line 1395, in _changed
    c._changed()
  File "/home/jales/software/miniconda/envs/ilastik-devel/ilastik-meta/lazyflow/lazyflow/slot.py", line 1395, in _changed
    c._changed()
  File "/home/jales/software/miniconda/envs/ilastik-devel/ilastik-meta/lazyflow/lazyflow/slot.py", line 1401, in _changed
    self._configureOperator(self)
  File "/home/jales/software/miniconda/envs/ilastik-devel/ilastik-meta/lazyflow/lazyflow/slot.py", line 1415, in _configureOperator
    self.operator._setupOutputs()
  File "/home/jales/software/miniconda/envs/ilastik-devel/ilastik-meta/lazyflow/lazyflow/operator.py", line 450, in _setupOutputs
    self.setupOutputs()
  File "/home/jales/software/miniconda/envs/ilastik-devel/ilastik-meta/ilastik/ilastik/applets/dataSelection/opDataSelection.py", line 282, in setupOutputs
    opReader.FilePath.setValue(datasetInfo.filePath)
  File "/home/jales/software/miniconda/envs/ilastik-devel/ilastik-meta/lazyflow/lazyflow/slot.py", line 110, in call_in_setup_context
    return func(self, *args, **kwargs)
  File "/home/jales/software/miniconda/envs/ilastik-devel/ilastik-meta/lazyflow/lazyflow/slot.py", line 1195, in setValue
    self._changed()
  File "/home/jales/software/miniconda/envs/ilastik-devel/ilastik-meta/lazyflow/lazyflow/slot.py", line 1401, in _changed
    self._configureOperator(self)
  File "/home/jales/software/miniconda/envs/ilastik-devel/ilastik-meta/lazyflow/lazyflow/slot.py", line 1415, in _configureOperator
    self.operator._setupOutputs()
  File "/home/jales/software/miniconda/envs/ilastik-devel/ilastik-meta/lazyflow/lazyflow/operator.py", line 450, in _setupOutputs
    self.setupOutputs()
  File "/home/jales/software/miniconda/envs/ilastik-devel/ilastik-meta/lazyflow/lazyflow/operators/ioOperators/opInputDataReader.py", line 158, in setupOutputs
    self.internalOperators, self.internalOutput = openFunc(filePath)
  File "/home/jales/software/miniconda/envs/ilastik-devel/ilastik-meta/lazyflow/lazyflow/operators/ioOperators/opInputDataReader.py", line 475, in _attemptOpenAsTiff
    opReader.Filepath.setValue(filePath)
  File "/home/jales/software/miniconda/envs/ilastik-devel/ilastik-meta/lazyflow/lazyflow/slot.py", line 110, in call_in_setup_context
    return func(self, *args, **kwargs)
  File "/home/jales/software/miniconda/envs/ilastik-devel/ilastik-meta/lazyflow/lazyflow/slot.py", line 1195, in setValue
    self._changed()
  File "/home/jales/software/miniconda/envs/ilastik-devel/ilastik-meta/lazyflow/lazyflow/slot.py", line 1401, in _changed
    self._configureOperator(self)
  File "/home/jales/software/miniconda/envs/ilastik-devel/ilastik-meta/lazyflow/lazyflow/slot.py", line 1415, in _configureOperator
    self.operator._setupOutputs()
  File "/home/jales/software/miniconda/envs/ilastik-devel/ilastik-meta/lazyflow/lazyflow/operator.py", line 450, in _setupOutputs
    self.setupOutputs()
  File "/home/jales/software/miniconda/envs/ilastik-devel/ilastik-meta/lazyflow/lazyflow/operators/ioOperators/opTiffReader.py", line 58, in setupOutputs
    if first_page.is_indexed:
  File "/home/jales/software/miniconda/envs/ilastik-devel/lib/python2.7/site-packages/tifffile.py", line 1919, in __getattr__
    raise AttributeError(name)
AttributeError: is_indexed

**Comment by Richard-Beck** [comment]
also get the same error message - cannot open project after upgrade to orc10. 

[ilastik_log.txt](https://github.com/ilastik/ilastik/files/534053/ilastik_log.txt)

**Comment by stuarteberg** [comment]
This should be fixed in our next release.  We reverted some of our TIFF-related code to an earlier version that doesn't suffer from this issue.

x-refs:
- https://github.com/ilastik/ilastik-build-conda/commit/59f77b571aea30b8dc4e8dcb75e027ec7e3557f8
- https://github.com/ilastik/lazyflow/commit/7630e0eb6ee0a7986dbd2aaf24b224b547172610

**Comment by cbecker** [comment]
Hi Stuart, is the next release out? I was having the same problem.

**Comment by stuarteberg** [comment]
Should be fixed in the current release (1.2.0)
